projects/conda-forge-simulation-stack.yml

```diff
@@ -0,0 +1,54 @@
+---
+name: Packaging the HEP simulation stack on conda-forge
+postdate: 2023-09-05
+categories:
+  - Analysis tools
+  - Open science
+durations:
+  - 3 months
+skillset:
+  - Python
+  - C++
+  - Docker
+status:
+  - Available
+project:
+  - IRIS-HEP
+experiments:
+  - ATLAS
+  - LHCb
+  - CMS
+  - ALICE
+  - EIC
+  - HLLHC
+program:
+  - IRIS-HEP fellow
+location:
+  - Remote
+commitment:
+  - Full time
+shortdescription: Package the HEP simulation tools for conda-forge
+description: >
+  One common toolchain used in high energy physics for simulation is:
+  [MadGraph5_aMC@NLO](https://launchpad.net/mg5amcnlo) to
+  [PYTHIA8](https://pythia.org/) to [Delphes](https://github.com/delphes/delphes).
+  Installing these tools can be challenging at times, especially for new users.
+  [Conda-forge](https://conda-forge.org/) allows for distribution of arbitrarily
+  complex binaries across multiple platforms though the conda/mamba/micromamba/pixi
+  package management ecosystem.
+  As [ROOT](https://anaconda.org/conda-forge/root/) has been successfully packaged and
+  distributed on conda-forge along with the
+  [PYTHIA8 Python bindings](https://anaconda.org/conda-forge/pythia8/) it should be
+  possible to package all the components of the HEP simulation stack and distribute
+  them on conda-forge.
+
+  However, the interconnected nature of some of these tools requires that multiple
+  dependencies are first packaged and distributed before the full stack can be.
+  This project would attempt to package as many of the dependencies of the HEP
+  simulation stack on conda-forge as possible starting with
+  [FastJet](http://fastjet.fr/), [LHAPDF](https://lhapdf.hepforge.org/),
+  [HepMC2](https://gitlab.cern.ch/hepmc/HepMC), and
+  [HepMC3](https://gitlab.cern.ch/hepmc/HepMC3).
```

Comment:
These two are already there:

* https://github.com/conda-forge/hepmc2-feedstock
* https://github.com/conda-forge/hepmc3-feedstock

Reply:
Amazing. :) 

```console
$ docker run --rm -ti mambaorg/micromamba:1.4.9-bullseye-slim
(base) mambauser@20be5bfb0b84:/tmp$ micromamba install --yes --channel conda-forge hepmc3
(base) mambauser@20be5bfb0b84:/tmp$ micromamba list hepmc3
List of packages in environment: "/opt/conda"

  Name    Version  Build       Channel    
────────────────────────────────────────────
  hepmc3  3.2.6    h93df44d_0  conda-forge
```

Reply:
Ah, but no Python bidings: https://github.com/conda-forge/hepmc3-feedstock/issues/16

That would be important to have.